spec/support/gitlab_helper.rb

```diff
@@ -27,19 +27,19 @@ def stub_request_source
 
       def stub_merge_request(fixture, slug, merge_request_id)
         raw_file = File.new("spec/fixtures/gitlab_api/#{fixture}.json")
-        url = "https://gitlab.com/api/v3/projects/#{slug}/merge_request/#{merge_request_id}"
```

Comment:
I expect people are using both 3.x and 4.x in production ( as we migrated Danger last week) - I found the issue, looks correct https://github.com/NARKOZ/gitlab/issues/229